src/components/MaskingInput.tsx

```diff
@@ -13,11 +14,9 @@ const Flex = styled.div`
   font-size: 16px;
   color: #000;
   cursor: pointer;
-  margin: 16px;
 `;
 const Dot = styled.div<IDotProps>`
-  width: 16px;
-  height: 16px;
+  ${({ size }) => `width: ${size}px; height: ${size}px;`}
```

Comment:
[Nit]

```suggestion
  ${({ size }) =>
    css`
      width: ${size}px;
      height: ${size}px;
    `}
```

별건 아니지만, styled-components에서 제공해주는 css 함수를 사용해줘도 좋을 것 같아요.

vscode에서 [vscode-styled-components](https://marketplace.visualstudio.com/items?itemName=styled-components.vscode-styled-components)을 사용하고 계시다면 오타가 날 가능성이 줄어든다고 생각해서요!

Reply:
이거 저번에도 한번 말씀해주신것 같네요 ㅠㅠ
추가했습니다!